fluentbit.tf

```diff
@@ -1,7 +1,7 @@
 locals {
   // optional FluentBit container for log aggregation
   fluentbit_container_defaults = {
-    name                   = "fluentbit"
+    name                   = var.firelens.container_name
     image                  = "${data.aws_caller_identity.current.account_id}.dkr.ecr.${data.aws_region.current.name}.amazonaws.com/ecr-public/aws-observability/aws-for-fluent-bit:2.29.0"
```

Comment:
we could [bump](https://gallery.ecr.aws/aws-observability/aws-for-fluent-bit) to 2.32 here as well

Reply:
```suggestion
    image                  = "${data.aws_caller_identity.current.account_id}.dkr.ecr.${data.aws_region.current.name}.amazonaws.com/ecr-public/aws-observability/aws-for-fluent-bit:2.32.0"
```